.github/workflows/lint.yml

```diff
@@ -43,9 +42,22 @@ jobs:
         id: check_changes
         run: echo "::set-output name=docs_only::${{ fromJSON(steps.changes.outputs.all_count) == fromJSON(steps.changes.outputs.docs_count) && fromJSON(steps.changes.outputs.docs_count) > 0 }}"
 
-      - name: Lint chart
+      - name: Set up Helm
+        if: ${{ steps.check_changes.outputs.docs_only != 'true' }}
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+
+      - name: Set up Python
         if: ${{ steps.check_changes.outputs.docs_only != 'true' }}
-        id: lint
-        uses: helm/chart-testing-action@v2.0.0
```

Comment:
This action is never allowed by INFRA settings and thus we always failed before. Fix on the fly.